docs/maps/indexing-and-styling-geojson-data.asciidoc

```diff
@@ -0,0 +1,257 @@
+steps to upload and index a GeoJSON File>> using the file
+`modified_flight_path.geojson`
+. Set *Layer name* to "Modified flight path".
+. Scroll down to the *Layer Style* section. On the map, this flight path nicely
+avoids the area of lightning strikes and should be the preferred route. Click
+*Border color* and change the color to a shade of green as a visual indicator
+that this will be the chosen flight path.
+. Again, feel free to change the *Border width* to a size that looks appropriate.
+. Click *Save & close*
+
+With both the original and the updated flight paths added, your map should now
+look something like this:
+
+[role="screenshot"]
+image::maps/images/fu_gs_flight_paths.png[]
+
+
+==== Save map
+Now that your map is complete, you'll want to save it so others can use it.
+
+. In the application toolbar, click *Save*.
+. Enter `Tutorial geojson upload` for the title.
+. Click *Confirm Save*.
++
+[role="screenshot"]
+image::maps/images/fu_gs_save_map.png[]
+
+[role="xpack"]
+=== Organizing layers and leveraging aggregations on indexed data
+Looking over the map, it's actually fairly complete and could be considered
+done, but there are a few more additions and tweaks that can be made to tell a
+better story with our newly indexed data.
+
+Looking at the `Lightning detected` layer, it's pretty clear where lightning has
+struck. What's a little less clear, is if there have been more lightning 
+strikes in some areas vs. others, or in other words, where the real lightning 
+hot spots are. One advantage of now having indexed `geo_point` data for the 
+lightning strikes is that we can perform aggregations on the data. 
+
+==== Add heatmap aggregation layer for lightning strikes
+. Below the map legend, click *Add layer*.
+. From the list of layer types, click *Grid aggregation*.
+. Since you've indexed `lightning_detected.geojson` using the index name & 
+pattern `lightning_detected`, that data is available to us as a `geo_point`
+aggregation. Go ahead and select the *Index pattern* `lightning_detected`
+. Next click *Show as* and select `heat map`.
+. Click *Add layer* to add the Heat map layer.
+. On the *Layer Settings* panel, go ahead and update the *Layer name* to be
+"Lightning intensity"
+. The remaining default settings are pretty good, but there are a couple other
+settings you might like to change. Under *Source Settings* > *Grid resolution*
+you can select from different heat map resolutions. The default "Coarse" looks
+pretty good, but feel free to select a different resolution.
+. Another setting that's worth playing around with is *Layer Style* >
+*Color range*. Again the default looks pretty good, but feel free to choose a
+different color range.
+. When you're finished modifying settings, click *Save & close*
+
+With your new lightning heat map layer added, your map should now look
+something like this:
+
+[role="screenshot"]
+image::maps/images/fu_gs_lightning_intensity.png[]
+
+
+==== Layer order
```

Comment:
Do you think it might make sense to add some of this info back in?  Right now, the tutorial just says to adjust the layers and zoom levels.  Maybe give some rationale as to why the user would want to do these things

Reply:
I added some more context for why a user might want to update layer styles, etc. with a couple more ideas.